content/en/docs/reference/node/kubelet-files.md

```diff
@@ -0,0 +1,138 @@
+---
+content_type: "reference"
+title: Local Files And Paths Used By The Kubelet
+weight: 42
+---
+
+The {{< glossary_tooltip text="kubelet" term_id="kubelet" >}} is mostly a stateless
+process running on a Kubernetes {{< glossary_tooltip text="node" term_id="node" >}}.
+This document outlines files that kubelet reads and writes.
+
+The kubelet typically uses the {{< glossary_tooltip text="control plane" term_id="control-plane" >}} as
+the source of truth on what needs to run on the Node, and the
+{{<glossary_tooltip text="container runtime" term_id="container-runtime">}} to retrieve
+the current state of containers. So long as you provide a _kubeconfig_ (API client configuration)
+to the kubelet, the kubelet does connect to your control plane; otherwise the node operates in
+_standalone mode_.
+
+On Linux the kubelet also relies on reading cgroups and various system files to collect metrics.
+
+There are also a few other files that are used by the kubelet as well as kubelet communicates using local Unix-domain sockets. Some are sockets that the
+kubelet listens on, and for other sockets the kubelet discovers them and then connects
+as a client.
+
+{{<note>}}
+
+This article lists path's as Linux paths, which maps to the Windows paths by adding a root disk `C:/` unless specified otherwise. For example, `/var/lib/kubelet/device-plugins` maps to `C:\var\lib\kubelet\device-plugins`.
```

Comment:
we could highlight this with a panel or some other UI functionality

Reply:
(nit)
```suggestion
This article lists paths as Linux paths, which map to the Windows paths by adding a root disk `C:\` in place of `/`
(unless specified otherwise).
For example, `/var/lib/kubelet/device-plugins` maps to `C:\var\lib\kubelet\device-plugins`.
```

Reply:
thank you! This is definitely better wording!